src/app/fyle/help/help.page.ts

```
@@ -37,12 +37,12 @@ export class HelpPage implements OnInit {
           this.orgUserService.getEmployeesByParams({
-            select: 'us_full_name,us_email',
-            ou_org_id: 'eq.' + eou.ou.org_id,
-            ou_roles: 'like.%' + 'ADMIN%',
-            ou_status: 'eq.' + '"ACTIVE"',
-            ou_id: 'not.eq.' + eou.ou.id,
-            order: 'us_full_name.asc,ou_id',
+            select: '(full_name,email)',
+            roles: 'like.%' + 'ADMIN%',
+            is_enabled: 'eq.true',
+            has_accepted_invite: 'eq.true',
+            id: 'neq.' + eou.ou.id,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)